### PR TITLE
Add troubleshooting guide for console to development docs

### PR DESCRIPTION
**Pull request by kschiffer**
#### Summary
This PR adds a console troubleshooting section to the development documentation. Hopefully, this will help developers to resolve common console issues.

#### Changes

- Add a console troubleshooting section to the development documentation

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, database and configuration, according to the stability commitments in `README.md`.
- [x] Testing: The changes are covered with unit tests. The changes are tested manually as well.
- [x] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Comment by htdvisser** [comment]
I'm not sure if this belongs in `DEVELOPMENT.md`

@johanstokking what do you think?

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -529,3 +529,70 @@ $ git push origin master
...
+### Console Troubleshooting
+
+#### Problem: Assets are not found
+The console will render a blank page and you can see backend logs like e.g.:
+```
+INFO Request handled                          duration=40.596µs error=error:pkg/errors/web:unknown (Not Found) message=Not Found method=GET namespace=web remote_addr=[::1]:50450 request_id=01DZ2CJDWKAFS10QD1NKZ1D56H status=404 url=/assets/console.36fcac90fa2408a19e4b.js
+```
+#### Possible causes
+##### Missing restart
+The stack has not been restarted after the console bundle has changed. In production mode, the stack will access the bundle via a filename that contains a content-hash, which is set during the build process of the console. The hash cannot be updated durig runtime and will take effect only after a restart.
```
Comment (rvolosatovs) [inline DEVELOPMENT.md]:
```suggestion
The stack has not been restarted after the console bundle has changed. In production mode, the stack will access the bundle via a filename that contains a content-hash, which is set during the build process of the console. The hash cannot be updated during runtime and will take effect only after a restart.
```

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -529,3 +529,70 @@ $ git push origin master
...
+
+##### Possible solution
+  1. Rebuild the console `mage js:clean js:build`
+  2. Restart the stack
+
+##### Mixing up production and development builds
+If you switch between production and development builds of the console, you might forget to re-run the build process and to restart the stack. Likewise, you might have arbitrary config options set that are specific to  a respective build type.
+
+##### Possible solution
+  1. Double check whether you have set the correct environment: `echo $NODE_ENV`
```
Comment (rvolosatovs) [inline DEVELOPMENT.md]: Some more context is required here - what are the possible values?

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -529,3 +531,70 @@ $ git push origin master
 ```
 
 After pushing the tag, our CI system will start building the release. When this is done, you'll find a new release on the [releases page](https://github.com/TheThingsNetwork/lorawan-stack/releases). After this is done, you'll need to edit the release notes. We typically copy-paste these from `CHANGELOG.md`.
+
+
+### Console Troubleshooting
+
+#### Problem: Assets are not found
```
Comment (neoaggelos) [inline DEVELOPMENT.md]: Perhaps add newlines here to adhere with the formatting of the rest of the document. Also for the other headings

File: DEVELOPMENT.md [hunk 4]
```diff
@@ -529,3 +531,70 @@ $ git push origin master
...
+The stack has not been restarted after the console bundle has changed. In production mode, the stack will access the bundle via a filename that contains a content-hash, which is set during the build process of the console. The hash cannot be updated during runtime and will take effect only after a restart.
+
+##### Possible solution
+  1. Restart the stack
+
+##### Accidentally deleted bundle files
+The bundle files have been deleted. This might happen e.g. when a mage target encountered an error and quit before running through.
+
+##### Possible solution
+  1. Rebuild the console `mage js:clean js:build`
```
Comment (neoaggelos) [inline DEVELOPMENT.md]: Maybe having `./mage` is better here (to adhere with the rest of the document as well). Also in other places below

File: DEVELOPMENT.md [hunk 5]
```diff
@@ -529,3 +531,70 @@ $ git push origin master
 ```
 
 After pushing the tag, our CI system will start building the release. When this is done, you'll find a new release on the [releases page](https://github.com/TheThingsNetwork/lorawan-stack/releases). After this is done, you'll need to edit the release notes. We typically copy-paste these from `CHANGELOG.md`.
+
+
+### Console Troubleshooting
+
+#### Problem: Assets are not found
+The console will render a blank page and you can see backend logs like e.g.:
```
Comment (johanstokking) [inline DEVELOPMENT.md]: Console

File: DEVELOPMENT.md [hunk 6]
```diff
@@ -529,3 +531,70 @@ $ git push origin master
...
+```
+INFO Request handled                          duration=40.596µs error=error:pkg/errors/web:unknown (Not Found) message=Not Found method=GET namespace=web remote_addr=[::1]:50450 request_id=01DZ2CJDWKAFS10QD1NKZ1D56H status=404 url=/assets/console.36fcac90fa2408a19e4b.js
+```
+#### Possible causes
+##### Missing restart
```
Comment (johanstokking) [inline DEVELOPMENT.md]: Insert empty lines, also below

File: DEVELOPMENT.md [hunk 7]
```diff
@@ -529,3 +531,70 @@ $ git push origin master
...
+The stack has not been restarted after the console bundle has changed. In production mode, the stack will access the bundle via a filename that contains a content-hash, which is set during the build process of the console. The hash cannot be updated during runtime and will take effect only after a restart.
+
+##### Possible solution
+  1. Restart the stack
```
Comment (johanstokking) [inline DEVELOPMENT.md]: The Things Stack

File: DEVELOPMENT.md [hunk 8]
```diff
@@ -529,3 +531,70 @@ $ git push origin master
...
+### Console Troubleshooting
+
+#### Problem: Assets are not found
+The console will render a blank page and you can see backend logs like e.g.:
+```
+INFO Request handled                          duration=40.596µs error=error:pkg/errors/web:unknown (Not Found) message=Not Found method=GET namespace=web remote_addr=[::1]:50450 request_id=01DZ2CJDWKAFS10QD1NKZ1D56H status=404 url=/assets/console.36fcac90fa2408a19e4b.js
+```
+#### Possible causes
+##### Missing restart
+The stack has not been restarted after the console bundle has changed. In production mode, the stack will access the bundle via a filename that contains a content-hash, which is set during the build process of the console. The hash cannot be updated during runtime and will take effect only after a restart.
```
Comment (johanstokking) [inline DEVELOPMENT.md]: The Things Stack

Comment (johanstokking) [reply DEVELOPMENT.md]: Console

File: DEVELOPMENT.md [hunk 9]
```diff
@@ -529,3 +531,70 @@ $ git push origin master
...
+
+##### Possible solutions
+- Re-establish a proper module link between the console and the JS SDK
+  - Run `mage js:cleanDeps js:deps`
+  - Check whether the `ttn-lw` symlink exists inside `node_modules` and whether it points to the right destination: `lorawan-stack/sdk/js/dist`
+    - If you have multiple ttn repos set up, `yarn link` might associate the JS SDK module with the SDK on another ttn repository
+  - Rebuild the console and (only after the build has finished) restart the stack
+
+#### Problem: The build crashes without showing any helpful error message
+#### Cause
```
Comment (johanstokking) [inline DEVELOPMENT.md]: Do these headings belong together?

File: DEVELOPMENT.md [hunk 10]
```diff
@@ -529,3 +531,70 @@ $ git push origin master
...
+```
+#### Possible causes
+##### Bundle using old JS SDK
+The bundle integrates an old version of the JS SDK. This is likely a caching/linking issue of the JS SDK dependency
+
+##### Possible solutions
+- Re-establish a proper module link between the console and the JS SDK
+  - Run `mage js:cleanDeps js:deps`
+  - Check whether the `ttn-lw` symlink exists inside `node_modules` and whether it points to the right destination: `lorawan-stack/sdk/js/dist`
+    - If you have multiple ttn repos set up, `yarn link` might associate the JS SDK module with the SDK on another ttn repository
```
Comment (johanstokking) [inline DEVELOPMENT.md]: "ttn repos" ?

File: DEVELOPMENT.md [hunk 11]
```diff
@@ -529,3 +531,94 @@ $ git push origin master
 ```
 
 After pushing the tag, our CI system will start building the release. When this is done, you'll find a new release on the [releases page](https://github.com/TheThingsNetwork/lorawan-stack/releases). After this is done, you'll need to edit the release notes. We typically copy-paste these from `CHANGELOG.md`.
+
```
Comment (neoaggelos) [inline DEVELOPMENT.md]: Single new-line here

**Pull request merged**